pkg/pma/util.c

```diff
@@ -0,0 +1,74 @@
+/// @file
+
+#include "util.h"
+
+#include <errno.h>
+#include <stdio.h>
+#include <string.h>
+#include <sys/types.h>
+#include <unistd.h>
+
```

Comment:
Need to define a single exported symbol for each of the inline functions defined in util.h debug builds break without this

```c
size_t
max(size_t a, size_t b);

size_t
min(size_t a, size_t b);

size_t
round_down(size_t x, size_t n);

size_t
round_up(size_t x, size_t n);
```

Reply:
Done.

Reply:
My bad - they were originally in there, but I gave @mcevoypeter some bad advice to remove them. I double-checked on the regular build, and tests that this worked, but not on the debug build.